src/Vulkan/Silk.NET.Vulkan/Vk.cs

```diff
@@ -67,67 +68,15 @@ public static Vk GetApi()
 
         public static unsafe Vk GetApi(ref InstanceCreateInfo info, out Instance instance)
         {
-            var ctx = new MultiNativeContext
-                (CreateDefaultContext(new VulkanLibraryNameContainer().GetLibraryName()), null);
-            var ret = new Vk(ctx);
-            ctx.Contexts[1] = new LamdaNativeContext
-            (
-                x =>
-                {
-                    if (x.EndsWith("ProcAddr"))
-                    {
-                        return default;
-                    }
-
-                    nint ptr = default;
-                    ptr = ret.GetInstanceProcAddr(ret.CurrentInstance.GetValueOrDefault(), x);
-                    if (ptr != default)
-                    {
-                        return ptr;
-                    }
-
-                    ptr = ret.GetDeviceProcAddr(ret.CurrentDevice.GetValueOrDefault(), x);
-                    return ptr;
-                }
-            );
-
-            fixed (InstanceCreateInfo* infoPtr = &info)
-            {
-                fixed (Instance* instancePtr = &instance)
-                {
-                    ret.CreateInstance(infoPtr, null, instancePtr);
-                }
-            }
-
+            var ret = GetApi();
+            instance = default;
```

Comment:
Is this necessary?

Reply:
no, just simplifying code?

Reply:
No as in do we have to initialize the instance variable? Surely this happens when we pass it along...

Reply:
I thought I had just carried that over